maud/Cargo.toml

```diff
@@ -18,6 +18,7 @@ default = []
 # Web framework integrations
 actix-web = ["actix-web-dep", "futures-util"]
 axum = ["axum-core", "http"]
+poem-3 = ["poem"]
```

Comment:
Can we remove this and name the feature `poem` instead (without the number)?

We only support the latest version of every framework, so the `3` is redundant.

Reply:
Sure! Thanks for taking a look.